pkgs/misc/vim-plugins/default.nix

```diff
@@ -941,6 +942,35 @@ rec {
 
   };
 
+  color_coded = buildVimPluginFrom2Nix { # created by nix#NixDerivation
+    name = "color_coded-2017-10-30";
+    src = fetchgit {
+      url = "https://github.com/jeaye/color_coded";
+      rev = "cb356b083362ea6f15bc690c5eb10b01326ad169";
+      sha256 = "0l46ar1b3vy7gbrq2ic734vvw7zcvw6ar78zyhwz55h2lxfh89aq";
+    };
+    dependencies = [];
+    buildInputs = [
+      stdenv
+      cmake
+      lua51Packages.lua
+      ncurses
+      zlib
+      llvm_39
+      llvmPackages.clang.cc
```

Comment:
I think this might be causing the darwin failure, since the stdenv there already uses clang.